pkg/jobs/transform_pulp_logs.go

```diff
@@ -70,48 +75,102 @@ func TransformPulpLogs(args []string) {
 		}
 		for i := 0; i < count; i++ {
 			newDate := date.AddDate(0, 0, i)
-			err := ProcessForDate(newDate)
+			log.Info().Msgf("Processing pulp logs for date %v", newDate)
+			success, messageCount, err := ProcessForDate(newDate)
 			if err != nil {
 				log.Error().Err(err).Msgf("Failed to process pulp logs for date %v", newDate)
+			} else if success && messageCount > 0 {
+				// Save the successful date to memo
+				err = db.Connect()
```

Comment:
do we need to try to connect to the database both in the loop and before (on line 59)?